packages/api/src/perma-cache/get.js

```diff
@@ -17,36 +16,28 @@ import { decodeKey } from './utils.js'
 export async function permaCacheListGet(request, env) {
   const requestUrl = new URL(request.url)
   const { searchParams } = requestUrl
-  const { limit } = parseSearchParams(searchParams)
+  const { before, size } = parseSearchParams(searchParams)
 
-  // Get user perma-cache entries by requested size recursively until requested limit or complete
-  const kvPrefix = `${request.auth.user.id}`
+  const sortBy = searchParams.get('sortBy') || 'Date'
+  const sortOrder = searchParams.get('sortOrder') || 'Desc'
```

Comment:
Why capitals for `Date` and `Desc`?

Reply:
These should be validated in `parseSearchParams`.

I don't think we should have `before` as it's not valid when `sortBy` is anything other than date. I would suggest something like:

* `page` - page number to fetch (default 0)
* `size` - page size (default 25)
* `sort` - sort field (default date)
* `order` - sort ordering (asc/desc, default asc)

You pagination header needs to be updated to always contain these.